src/extensions/String.ts

```diff
@@ -3,14 +3,15 @@
 interface String {
   firstAndLastCharacters(amount: number): string
   trimLeftByPattern(pattern: string): string
+  padEnd(targetLength: any, padString: any): string
```

Comment:
Does using' number' and 'string' for the argument types cause a problem?

Reply:
There is logic in the function to handle if the targetLength passed in isn't a number, that's why I didn't use number here. So we would have to remove some of the tests if we did add the types.